modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -67,39 +68,48 @@ If you want to create virtual function on the default network interface, ensure
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManaged` to `true` to allow the SR-IOV Network Operator to configure and allocate a subset of virtual functions from
```

Comment:
The first sentence says the Operator configures a subset of VFs when `externallyManaged=true`. The second sentence says that the Operator configures the VFs when `externallyManaged=false`.

When `externallyManaged=true`, the Operator will not allocate or configure the VFs. The Operator can use the VFs to attach them to Pods but not manage them (i.e. create/allocate and configure)

Reply:
I thought the key point was that with this set to true the SR-IOV Network Operator is can configure a subset of "all" VF whereas with it set to false (think default) the SR-IOV Network Operator does not have that granular control and configures "all"

Reply:
On the contrary. The parameter is called `externallyManaged` which hints that VFs are not to be managed by the Operator. The default value is `false`, yes.

Reply:
Please consider the following suggestion:

"Set `externallyManaged` to `true` to allow the SR-IOV Network Operator to use all or a subset of externally managed virtual functions and attach them to Pods. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs."

Reply:
Thank you, @cgoncalves "to use **all** or a subset of externally managed virtual functions" seems like a important distinction which I must have missed.

Reply:
@cgoncalves  You are absolutely right:
* If externallyCreated is false (default) - SR-IOV must create/recreate the VFs on its own.
* if externallyCreated is true - SR-IOV should use VFs created earlier.

Reply:
@kquinn1204
 BTW This line is only relevant if externallyCreated is true. 

> Before applying the policy, it is necessary to create the VFs first; otherwise, the webhook will block the request.

Reply:
@evgenLevin I made that clarification thanks